src/compojure/api/api.clj

```diff
@@ -50,14 +52,19 @@
   [& body]
   (let [[options handlers] (common/extract-parameters body false)
         options (rsc/deep-merge api-defaults options)
-        handler (apply c/routes (concat [(swagger/swagger-routes (:swagger options))] handlers))
+        raw (:raw options)
+        swagger-ui (-> (swagger/swagger-routes (:swagger options))
+                       (cond-> raw ;; if the api won't be wrapped, swagger-ui still needs to be
```

Comment:
why does it have to be wrapped? feels odd if the swagger routes would have different format-settings.